### PR TITLE
Shared web credentials support

### LINKED ISSUE #37: Shared Web Credentials support?
https://developer.apple.com/reference/security/1654440-shared_web_credentials

### PR DESCRIPTION
**Pull request by oblador**
Closes #37

Note when testing: this is using ES2016 exports, so make sure to change import statement.

### CODE REVIEW

**Comment by messense** [comment]
Great job!

Can you also add [SecAddSharedWebCredential](https://developer.apple.com/reference/security/1617986-secaddsharedwebcredential?language=objc) for [creating/changing/deleting shared web credentials](https://developer.apple.com/reference/security/1654440-shared_web_credentials)?

**Comment by messense** [comment]
I tested the code, it works like charm.

**Comment by messense** [comment]
Will this be merged soon?

**Comment by SudoPlz** [comment]
That looks good! When will it be merged?

File: README.md [hunk 1]
```diff
@@ -43,10 +43,10 @@ pod 'RNKeychain', :path => 'node_modules/react-native-keychain'
 See `KeychainExample` for fully working project example.
 
 ```js
-var Keychain = require('react-native-keychain');
+import { * as Keychain} from 'react-native-keychain';
```
Comment (oblador) [inline README.md]: Yes it should, good catch 👍

**Comment by oblador** [comment]
@messense Setting added and merged 👍

**Comment by oblador** [comment]
Released in [1.0.0](https://github.com/oblador/react-native-keychain/releases/tag/v1.0.0).

**Comment by messense** [comment]
@oblador Can you release a 0.3.3 version without the RN 0.40 header changes commit please? Many people like me would be stuck with RN <= 0.39 for some time.

**Comment by SudoPlz** [comment]
Any idea when Android will be supported?
Thanks..!

**Pull request merged**